zmon-common/src/main/java/org/zalando/zmon/domain/CheckDefinition.java

```diff
@@ -88,6 +90,9 @@
     @DatabaseField
     private DefinitionRuntime runtime;
 
+    @Transient
+    private Tier tier = Tier.EMPTY;
```

Comment:
And once again (are we ever gonna end this discussion?). There's not possible to have an empty tier. If it's not defined, then it's "other". Only 3 possible values of this value - critical, important and other. No "empty", no "important" and no `null`

Reply:
We need somehow notify the frontend, that there is no entity with tiers at all. 
Because it is a OSS, we have to car about users outside of Zalando. And in case there are no entity with tiers, on the frontend we are hiding any trace of tier. So, the users won't see any "empty", "normal" or whatever.

Reply:
why don't we keep this field `null` then?

Reply:
I don't think there's anything wrong with showing `other` if no tier entity is set

Reply:
Done